cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -0,0 +1,95 @@
+# HTTP Cache configuration.
+#
+# See cdo-varnish/README.md for more information on the configuration format.
+class HttpCache
+  STATIC_ASSETS = {
```

Comment:
Evaluation stops with the first matching path. I'll make this explicit in the documentation.